### ISSUE TITLE
Error in safari mobile 10.3

### ISSUE THREAD

**Issue by felipeegasneris** [matthewp/haunted#92]
Hi, I'm building a component and testing it on safari mobile 10.3 and I getting this error: `TypeError: The Element.shadowRoot getter can only be used on instances of Element` on this line:` this._container = new Container(renderer, this.shadowRoot, this);` do you know why is this error? I need to make works haunted on Safari 10.3, please help me :(

**Comment by matthewp** [comment]
Thanks for reporting. We'll figure it out. Sounds like `this` is not an element. That's odd. What is `this`? there, can you inspect that?

**Comment by felipeegasneris** [comment]
inspecting this, I get: `this: "TypeError: The Node.ownerDocument getter can only be used on instances of Node"`

**Comment by felipeegasneris** [comment]
any clue on that @matthewp ?

**Comment by matthewp** [comment]
Can you give me a minimal example so that I can take a look? I don't think that version of Safari supports modules so my examples I have don't run on it. I guess you probably need a bundled app.  If you can give me a minimal example I'll try to debug it.

**Comment by matthewp** [comment]
ping @felipeegasneris

**Comment by cristinecula** [comment]
I ran into the same issue as @felipeegasneris. I created this reproduction branch: https://github.com/Neovici/cosmoz-image-viewer/tree/haunted-bug-repro

Just do `npm start` and open http://localhost:8080/dist/index.html in Safari 10.

**Comment by matthewp** [comment]
Cool, thanks for the reproduce!

**Comment by felipeegasneris** [comment]
:ping:

**Comment by felipeegasneris** [comment]
any clue on this?

**Comment by matthewp** [comment]
No, sorry. It's unlikely I'll personally be able to get to this any time soon. I would love a PR that fixes it.

**Comment by cletusw** [comment]
This looks to be fixed. I just tested via iOS Simulator -- I can repro per the [above steps](https://github.com/matthewp/haunted/issues/92#issuecomment-528222917) on iPhone 7 in iOS 10.3.1, but no error & page w/ image loads fine in 11.3, 12.4, 13.5 (iPhone 8).
